src/ansys/dpf/core/data_tree.py

```diff
@@ -482,6 +482,16 @@ def get_as(self, name, type_to_return=types.string):
             out = DataTree(data_tree=obj, server=self._server)
         return out
 
+    def get_attributes(self):
```

Comment:
@jose-henriquezroa could you please add a dosctring and in this docstring precise whether sub attributes are returned or not (whether this function is recursive basically?))

To comply with our other python classes I would make this function a property instead, what do you think?

Reply:
@cbellot000 I will add the docstring and add an option for recursively fetching attributes from subtrees as well. I think the option to do that will be very useful indeed.